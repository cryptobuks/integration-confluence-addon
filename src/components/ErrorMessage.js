import React from 'react'

const ErrorMessage = ({url}) => (
  <div className="aui-message aui-message-error">
    <p>
      <strong>Error!</strong>
    </p>
    <p>{ decodeURIComponent(url) } is not valid or password protected.</p>
  </div>
)

export default ErrorMessage
