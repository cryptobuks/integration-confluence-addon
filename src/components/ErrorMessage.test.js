import React from 'react'
import renderer from 'react-test-renderer'
import ErrorMessage from './ErrorMessage'

it('renders correctly', () => {
  const tree = renderer.create(
    <ErrorMessage url="http://badlink" />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
