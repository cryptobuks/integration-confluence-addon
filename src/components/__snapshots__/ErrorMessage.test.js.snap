// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="aui-message aui-message-error"
>
  <p>
    <strong>
      Error!
    </strong>
  </p>
  <p>
    http://badlink
     is not valid or password protected.
  </p>
</div>
`;
